### DISCUSSION TITLE
Opening a pdf link shows page not found

### DISCUSSION THREAD

**Discussion by rafaelrdealmeida** [facebook/docusaurus#4637] [Q&A] [answered]
I don't know if it's a bug or if I'm doing something wrong (probably, because I think this was resolved in #3337). 

I'm not able to open pdf links hosted on docusaurus

In editing the pdf is opened, but in the build it is not

Could someone help me?

see:

(1) click on the pdf download icon on the following page:

`https://labriunesp.org/cadernos/labri-n0002`

(2) is directed to the page of the pdf file and page not found

`https://labriunesp.org/arquivos/2020-04-30-labri-002.pdf`

In

`https://gitlab.com/unesp-labri/sites/labri/-/blob/main/website/src/theme/BlogPostPage/index.js`

set `download_pdf `on **line 33** and reference on **line 62**

download_pdf comes from the .md header. See: 

`https://gitlab.com/unesp-labri/sites/labri/-/blob/main/website/cadernos/2020-04-30-labri-002.md`

best

**Comment by slorber** [comment, marked as answer]
After a build, you probably don't have a file at `website/build/arquivos/2020-04-30-labri-002.pdf` 

So your host can't serve it, and it defaults to the 404 page of Docusaurus.

How do you ensure that this file is available in the build folder? It is your responsibility to make it available. You can put it in the `static` folder

Docusaurus is not aware of your custom `download_pdf` frontmatter, and won't do anything about it to make the pdf available.

**Reply by rafaelrdealmeida** [reply]
Thanks!
